lib/node_modules/@stdlib/repl/lib/auto_close_pairs.js

```diff
@@ -65,8 +65,6 @@ function AutoCloser( rli ) {
 *
 * @name beforeKeypress
 * @memberof AutoCloser.prototype
-* @param {string} data - input data
```

Comment:
@Snehil-Shah Why were these changes made? It is not uncommon for us to document parameters even if we don't use them in functions, as this allows for future dev reference. E.g., if we end up needing to modify `beforeKeypress` in the future to use `key`, we can reference the JSDoc, rather than track down the downstream callsite.

Reply:
I apologize, I thought it was a mistake. Will revert.